scripts/travis/install.sh

```diff
@@ -12,12 +12,9 @@ echo "Install Python dependencies"
 ./peep.sh install -r requirements/default.txt
 echo
 
-# Installing dependencies for smoke tests
-if [[ $TEST_SUITE == "smoke" ]]; then
-  virtualenv venv_smoketests
-  source venv_smoketests/bin/activate
-  pip install -r smoketests/requirements.txt
-  deactivate
+# Installing dependencies for UI tests
+if [[ $TEST_SUITE == "ui" ]]; then
+  ./peep.sh install -r requirements/test.txt
```

Comment:
In future the requirements may be incompatible and that may raise issues. I strongly believe that it will be better for long run if this requirements are installed in different virtualenv and so we can debug it easily in future if any failure.